testsuite/tests/warnings/deprecated_module_assigment.compilers.reference

```diff
@@ -1,72 +1,72 @@
 File "deprecated_module_assigment.ml", line 17, characters 33-34:
-Warning 3: deprecated: x
+Alert deprecated: x
 DEPRECATED
-           File "deprecated_module_assigment.ml", line 12, characters 2-41:
-           Definition
-           File "deprecated_module_assigment.ml", line 17, characters 15-26:
-           Expected signature
+                  File "deprecated_module_assigment.ml", line 12, characters 2-41:
```

Comment:
Not sure I understand this whitespace change...

Reply:
Ping @Armael

Reply:
That's because the whole error/warning/alert message format box is now aligned on the right of `Error: `/`Warning X:`/`Alert foo: `. For example:

- on 4.07:
```
# module X : sig val x : int [@@deprecated] end = struct let x = 7 end;;
# module Y : sig val x : int end = X;;
Characters 33-34:
Warning 3: deprecated: x
  Characters 16-42:
  Definition
  Characters 15-26:
  Expected signature
```

- on trunk:
```
# module X : sig val x : int [@@deprecated] end = struct let x = 7 end;;
# module Y : sig val x : int end = X;;
Warning 3: deprecated: x
           Line 1, characters 15-41:
           Definition
           Line 1, characters 15-26:
           Expected signature
```

- on this pr, format box is the same but there's more text on the left:
```
# module X : sig val x : int [@@deprecated] end = struct let x = 7 end;;
# module Y : sig val x : int end = X;;
Alert deprecated: x
                  Line 1, characters 15-41:
                  Definition
                  Line 1, characters 15-26:
                  Expected signature
```
-----

I'm actually not really convinced by the current alignment setup (although I'm responsible for it); this PR gives an extra argument for changing it (since we now can have arbitrary long text -- the name of the alert -- printed at the left of the box).

Reply:
(fixed in #2035)

Reply:
Thanks.  I've already rebased this PR!